nomic/project.py

```diff
@@ -963,11 +967,62 @@ def get_projection_id():
 
     def __repr__(self):
         m = self.meta
-        return f"Nomic project: <{m}>"
+        return f"AtlasProject: <{m}>"
```

Comment:
I will add a _repr_html_ method for Ipython.

Reply:
Can we somehow detect if a user is in a jupyter notebook and just do the right thing instead?

Reply:
https://stackoverflow.com/questions/15411967/how-can-i-check-if-code-is-executed-in-the-ipython-notebook

we can